share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,32 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "piratetalk",
+
+            // Customize these properties
+            name: "Talk like a Pirate!",
```

Comment:
This is a Spice, so those are still required :)

Only Goodies don't need them for now.